### ISSUE TITLE
Google Fonts Knowledge *Point size*: correction needed and possible expansion

### ISSUE THREAD

**Issue by RuixiZhang42** [google/fonts#4491]
Quote https://fonts.google.com/knowledge/glossary/point_size

> The original point measured 72.289 parts per inch, but has been rounded to simply 72 since the 1990s, in line with the 72 dpi (dots per inch) of the displays of the first Macintosh computers.

This has two major issues:
1. The number “72.289” is misinformed. Correction is needed.
2. It doesn’t acknowledge other point systems (e.g., the Didot point of the Europeans), nor does it acknowledge typesetting software that follow tradition (e.g., TeX). Possible expansion is needed.

“The original point” in the quote is really just “the original (agreed) *American* point”, established by the Type Founders Association of the US in the 1890s. The US national standard *defines* one point to be *exactly* 0.013837 inches (NIST Handbook&nbsp;44, Appendix&nbsp;C). This makes a point measures 1/0.013837≈72.2700007 parts per inch. I have no idea where that 72.289 figure comes from…

TeX follows this tradition by defining its point to be exactly 72.27 parts per inch. The rounding of a point to 72 parts per inch appeared in the late 1970s (not 1990s) with the introduction of PostScript from Adobe. TeX also supports this bigger point.

An even bigger point was developed in Europe, which was based on the French inch (cue the Napoleon height jokes). A Didot point measures approximately 67.5583 parts per (American) inch.

I think it’s worth providing *accurate* definitions. It’s also worth acknowledging *world-wide* differences.

**Comment by thlinard** [comment]
Indeed… never heard of a point worth 72,289 parts per inch. After the first attempts by Sébastien Truchet and Pierre-Simon Fournier, it was indeed François Ambroise Didot who in 1783 defined the value of the point, i.e. exactly 1/72 of the French inch (a little bigger than the English inch: one-sixth of the _pied du roi_).

There was also the attempt at standardization by the Imprimerie Nationale around 1810, by creating a millimetric point, but this point is, to my knowledge, only used for the Didot millimétrique, an exclusive typeface of the Imprimerie Nationale.

The pica point is equal to 1/72,27 English inch.

The DTP point is equal to 1/72 English inch.

In the International System of Units:

1 Didot point = 0.3759 mm
1 millimetric point = 0.4000 mm
1 pica point = 0.3515 mm
1 DTP point = 0.3528 mm

French readers can learn more here: https://jacques-andre.fr/fontex/point-typo.pdf. English readers can read https://typefoundry.blogspot.com/2008/04/type-bodies-compared.html.
